variables.tf

```diff
@@ -154,6 +154,34 @@ variable "rhcos_kernel_options" {
     default     = []
 }
 
+variable "sysctl_tuned_options" {
+    description = "Set to true to apply sysctl options via tuned operator. Default: false"
+    default     = false
+}
+
+variable "sysctl_options" {
+    description = "List of sysctl options to apply."
+    default     = ["kernel.shmmni = 16384", "net.ipv4.tcp_tw_reuse = 1"]
```

Comment:
I think we should avoid defaults here.  We can instead provide these as examples